onnxscript/optimizer/__init__.py

```diff
@@ -35,6 +35,12 @@ def optimize(model: ir.Model, *args, **kwargs) -> ir.Model:
         return legacy_optimizer.optimize(model, *args, **kwargs)
 
 
+def inline(model: ir.Model) -> None:
+    """Inline all function calls (recursively) in the model."""
+    if model.functions:
+        onnxscript.ir.passes.common.inliner.InlinePass()(model)
```

Comment:
I wonder if we should do this to all the passes that are not applied by default? What is the pass API we have in mind?

Reply:
Do you mean creating a function for them? I think only for convenience. I imagine the pass api is how we currently do in `optimize()`, with a pass manager that's driving them. A little bit like how you can compose nn.Module inside nn.Module, we can do the same with passes too.

Reply:
OK.

I found another InlinePass under_inliner.py. Should we clean that up?

Reply:
Let me check, thanks

Reply:
I think this is the only one after this PR? Could you confirm?